src/mlagility/analysis/analysis.py

```diff
@@ -109,16 +109,11 @@ def call_benchit(
                 inputs[all_args[i]] = args[i]
     model_info.inputs = inputs
 
-    cache_dir = (
-        filesystem.DEFAULT_CACHE_DIR
-        if tracer_args.cache_dir is None
-        else tracer_args.cache_dir
-    )
```

Comment:
fyi @danielholanda I got rid of this because analysis.py being able to change the cache_dir is scary, in the context of benchmark_script.py also being able to change the cache dir.